packages/ui-kit-chat/src/theme/themes/terminal.css

```
@@ -0,0 +1,189 @@
+
+  --pw-r-sm: 4px;
+  --pw-r-md: 8px;
+  --pw-r-lg: 12px;
+  --pw-r-pill: 999px;
+
+  --pw-shadow: 0 2px 10px rgba(0, 0, 0, 0.5);
+  --pw-shadow-lg: 0 16px 44px rgba(0, 0, 0, 0.6);
+  --pw-shadow-hover: 0 16px 44px rgba(0, 0, 0, 0.6);
+
+  --pw-font:
+    'Source Sans 3', ui-sans-serif, system-ui, -apple-system, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)